Automattic/vip-cli
# PIE-2632: fix site-type validation for sql file

## Description

Cleans up site-type validation where site type might mismatch from the SQL dump file provided.

## Steps to Test

Example:

1. Check out PR.
1. Execute an import into a single site with a multisite table SQL dump --> result is failure
2. Execute an import into a multisite with a single site SQL dump --> result is failure



---
_This PR was started by: [git wf pr](https://github.com/groupon/git-workflow/releases/tag/v1.3.2)_
